applications/FSIapplication/custom_utilities/mvqn_recursive_convergence_accelerator.hpp

```diff
@@ -434,14 +443,15 @@ class MVQNRecursiveJacobianConvergenceAccelerator: public ConvergenceAccelerator
     {
         Parameters mvqn_recursive_default_parameters(R"(
         {
-            "solver_type" : "MVQN",
+            "solver_type" : "MVQN_recursive",
             "w_0"         : 0.825,
             "buffer_size" : 10
         }
         )");
 
         rConvAcceleratorParameters.ValidateAndAssignDefaults(mvqn_recursive_default_parameters);
 
+        mColumnCutOffRelTol = 1e-8;
```

Comment:
shouldn't this be in the parameters with a default value?

Reply:
Should be. Added to both constructors.